test/controllers/IndividualBeneficiaryAddressUKYesNoControllerSpec.scala

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import base.SpecBase
+import forms.IndividualBeneficiaryAddressUKYesNoFormProvider
+import models.{FullName, NormalMode, UserAnswers}
+import navigation.{FakeNavigator, Navigator}
+import pages.{IndividualBeneficiaryAddressUKYesNoPage, IndividualBeneficiaryNamePage}
+import play.api.inject.bind
+import play.api.libs.json.{JsBoolean, Json}
+import play.api.mvc.Call
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import views.html.IndividualBeneficiaryAddressUKYesNoView
+
+class IndividualBeneficiaryAddressUKYesNoControllerSpec extends SpecBase {
+
+  def onwardRoute = Call("GET", "/foo")
+
+  val formProvider = new IndividualBeneficiaryAddressUKYesNoFormProvider()
+  val form = formProvider()
+  val index: Int = 0
+  val name = FullName("first name", None, "Last name")
+
+  lazy val individualBeneficiaryAddressUKYesNoRoute = routes.IndividualBeneficiaryAddressUKYesNoController.onPageLoad(NormalMode, index).url
+
+  "IndividualBeneficiaryAddressUKYesNo Controller" must {
+
```

Comment:
Do you require a unit test for when the required answer is not present in mongo?

Reply:
Yes, added.